packages/react-ui-map/src/index.js

```diff
@@ -1,2 +1,4 @@
 export { default } from './Mapbox'
 export { default as Mapbox } from './Mapbox'
+export { default as MapSource } from './Mapbox/MapSource'
```

Comment:
I think we should probably change the names to `Source` and `Layer` for the filenames.  This is kinda nitpicky so it's up to what others ideas are.